### PR TITLE
Change adobeanalytics amp-analytics codepath

### PR DESCRIPTION
**Pull request by jmadler**
cc @cramforce

### CODE REVIEW

File: extensions/amp-analytics/0.1/vendors.js [hunk 1]
```diff
@@ -442,3 +442,5 @@ export const ANALYTICS_CONFIG = {
 ANALYTICS_CONFIG['infonline']['triggers']['pageview']['iframe' +
 /* TEMPORARY EXCEPTION */ 'Ping'] = true;
 
+ANALYTICS_CONFIG['adobeanalytics']['triggers']['pageview']['iframe' +
```
Comment (cramforce) [inline extensions/amp-analytics/0.1/vendors.js]: This breaks existing users. Lets rename the request, so users can migrate to the iframe pageview ping if they want to.

**Comment by cory-work** [comment]
@jmadler 

What about adding something like this:
`
    'triggers': {
      "pageLoad": {                                                                                                                                                                                                                                                                                                         
        "on": "visible",
        "request": "iframeMessage",
        "iframePing": true
      }   
    }
`

I've tested with this and it seems to work pretty well

**Comment by cramforce** [comment]
@cory-work +1 That was my suggestion.

**Comment by cory-work** [comment]
@cramforce @jmadler 

Do you guys want me to make a pull request or do you want to update this one?

**Comment by cramforce** [comment]
Not sure about @jmadler's timing. Would be great if you could take it on!

**Comment by jmadler** [comment]
Go for it!  I'll close this one.

**Comment by cory-work** [comment]
@jmadler & @cramforce 

I'm on it. I'll submit something soonish

**Pull request closed without merging**